templates/address-space-controller/020-ConfigMap-address-space-definitions.yaml

```diff
@@ -84,10 +84,17 @@ data:
             authSslProfile: auth_service_ssl
           }
 
+          listener {
+            host: 127.0.0.1
```

Comment:
This won't work if for any reason ipv4 is not used (localhost should resolve to either 127.0.0.1 or ::1). However that seems a fringe concern.